lib/build.gradle

```diff
@@ -8,6 +8,7 @@ dependencies {
     testImplementation 'org.slf4j:slf4j-jdk14:2.0.3'
     testImplementation 'org.openapitools.empoa:empoa-simple-models-impl:2.1.0'
     testImplementation 'nl.jqno.equalsverifier:equalsverifier:3.10.1'
+    implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0'
```

Comment:
Please keep the implementation near the other implementation. I like to keep `implementation` and `testImplementation` in separated blocks :)